Financials/_posts/Quarterly-Report/2021-Q2/2021-09-11-2021-Q2.md

```diff
@@ -4,389 +4,230 @@ title:  "Yearn Finance 2021 Q2 Quarterly Report"
-  entire loan related to this incident has been paid off from protocol earnings.
-- Yearn still has some outstanding debt owed to MakerDao, but possesses enough cash to
-  pay this debt at any time.
-- The photo presented at left is as of the date of this publication and not reflected of end
-  of month balance on June 2021 (Q2 closing).
+- Los usuarios de bóvedas v1 experimentaron un evento adverso a principios del primer trimestre que resultó en una pérdida de $11 millones. Yearn cubrió esta pérdida y reembolsó a los usuarios obteniendo un préstamo de MakerDao. La totalidad del préstamo relacionado con este incidente se ha pagado con los ingresos del protocolo.
+- Yearn todavía tiene una deuda pendiente con MakerDao, pero posee suficiente efectivo para pagar esta deuda en cualquier momento.
+- La foto que se presenta a la izquierda corresponde a la fecha de esta publicación y no refleja el balance de fin de mes en junio de 2021 (cierre del segundo trimestre).
 
-# Yearn _Total Value Locked exceeded $5.0b in June_, and as of August 19
+# El _valor total bloqueado de Yearn superó los $5.0 mil millones en junio_, y al 19 de agosto
 
 ![](page17.png)
 
-- There has been phenomenal growth in Total Value Locked (TVL) across all Yearn products
-  in Q1 2021 and continuing into Q2 2021. In June TVL peaked at $5.0b, slightly retraced in
-  July, but has since rebounded in August. ,
-- The majority of our TVL is derived from our v2 yVaultproduct line. In late May and
-  throughout June we transitioned a significant portion of our v1 vaults to v2 vaults. We no
-  longer support v1 vaults and encourage all depositors to migrate to v2 vaults.
-- Iron Bank is Yearn’sin-house lending platform that is in collaboration with C.R.E.A.M.
-  Finance. We continued to see growth in Iron Bank TVL from Q1 to Q2.
-- It is worth noting that as of August 19th, 2021, **Yearn has over $4.6b in TVL without any
-  token subsidies** or other incentives typically offered by competing protocols in DeFi.
-- _Note: The chart presented at left is as of August 19th, 2021._
-- Source: Github: [yearn/yearn-exporter](https://github.com/yearn/yearn-exporter)
-
-# We continue to make improvements to our financial dashboard displaying transactions, an Income Statement, and a Balance Sheet.
+- En el primer trimestre de 2021 se ha producido un extraordinario crecimiento del valor total bloqueado (VTB) en todos los productos de Yearn, y continuó en el segundo trimestre de 2021. En junio, el VTB alcanzó un máximo de $5.0 mil millones, ligeramente retrocedido en julio, pero desde entonces se ha recuperado en agosto. 
+- La mayor parte de nuestro VTL procede de nuestra línea de productos yVault v2. A finales de mayo y durante todo el mes de junio realizamos la transición de una parte importante de nuestras bóvedas v1 a las v2. Ya no admitimos bóvedas v1 y alentamos a todos los depositantes a migrar a bóvedas v2.
```

Comment:
AS-IS: La mayor parte de nuestro VTL procede de nuestra línea de productos yVault v2. A finales de mayo y durante todo el mes de junio realizamos la transición de una parte importante de nuestras bóvedas v1 a las v2. Ya no admitimos bóvedas v1 y alentamos a todos los depositantes a migrar a bóvedas v2.

TO-BE: La mayor parte de nuestro VTB procede de nuestra línea de productos yVault v2. A finales de mayo y durante todo el mes de junio realizamos la transición de una parte importante de nuestras bóvedas v1 a las v2. Ya no admitimos bóvedas v1 y alentamos a todos los depositantes a migrar a bóvedas v2.

Reply:
fixed